### PR TITLE
feat: add --fixable option with CLI parsing

### LINKED ISSUE #2: Do not auto-disable eslint issues which can be fixed
Currently, this CLI disables all warnings and errors in the source directory. If ESLint can fix something with the `--fix` option, this CLI can have an option to ignore such issues.

### PR DESCRIPTION
**Pull request by AndersDJohnson**
Adds a new option `--fixable` which will opt-in to disabling any "fixable" ESLint rules (with their `eslint --fix` option), which are now otherwise skipped (because it does not make sense to disable fixable rules by default when they can be corrected automatically).

Introduces the popular `yargs` library to parse this and future CLI arguments.

Fixes #2.

### CODE REVIEW

File: bin/index.js [hunk 1]
```diff
@@ -41,14 +47,18 @@ const writeModifiedRules = ruleIdsArr => {
  * @param {array} results - Array of files with ESLint report for each file
  */
 const disableESLintIssues = results => {
+  const includeFixable = argv.fixable;
```
Comment (AdarshKonchady) [inline bin/index.js]: Can we call this option `disable-fixable` instead (for both argument and the constant)? 

```suggestion
  const ignoreFixable = argv['ignore-fixable'];
```

Positive check here 
https://github.com/akonchady/disable-eslint-issues-cli/pull/4/files#diff-9ea21889e9fbc7d82e7dabb0fb280808R61

would be easier to grok as well.

**Review by AdarshKonchady — CHANGES_REQUESTED** [review]
Thanks for the PR ! 

I got rid of `package-lock.json` in a separate commit to avoid multiple lock files. Looks like we need to clean that up here. Wondering how `yarn.lock` works with GitHub security alerts though.

**Comment by AdarshKonchady** [comment]
Fixes #2

File: bin/index.js (hunk 1 above)
Comment (AndersDJohnson) [reply bin/index.js]: @akonchady We could. Which do you think makes more sense as a default? I felt like most users would not want to dirty up their code with disable comments for fixable rules, so I decided we should not disable fixable rules by default, but if a user wants it, we would require positive opt-in with the `--fixable` flag (or could have named it `--include-fixable`). What do you think?

**Comment by AndersDJohnson** [comment]
@akonchady I've fixed the conflicts.

File: package-lock.json [hunk 2]
```diff
@@ -0,0 +1,1160 @@
+{
```
Comment (AdarshKonchady) [inline package-lock.json]: You can delete this file now that there's `yarn.lock` in the repo.

Comment (AdarshKonchady) [reply package-lock.json]: Also, looks like you need to remove `modifiedRuleIds.txt` file as well.

File: bin/index.js (hunk 1 above)
Comment (AdarshKonchady) [reply bin/index.js]: Fair point. `--include-fixable` sounds good to me 👍

File: package-lock.json (hunk 2 above)
Comment (AndersDJohnson) [reply package-lock.json]: @akonchady Done, removed and merged with `master`.

File: bin/index.js (hunk 1 above)
Comment (AndersDJohnson) [reply bin/index.js]: @akonchady I have updated the option to `--include-fixable` (aliased as `-f`). I have added description text for the new `--help` flag that we support now that this moves us to use `yargs` for command-line parsing. I've updated the README with these new features and options.

**Review by AdarshKonchady — APPROVED** [review]
👍

**Pull request merged**